digiwf-apps/packages/apps/digiwf-tasklist/src/api/tasks/tasksApiCalls.ts

```diff
@@ -1,38 +1,209 @@
 import {
-  Configuration,
+  CompleteTO, DocumentRestControllerApiFactory,
   FetchUtils,
+  HumanTaskDetailTO,
+  Configuration,
   HumanTaskRestControllerApiFactory,
-  PageHumanTaskTO
+  PageHumanTaskTO, SaveTO, StatusDokumentTO
 } from "@muenchen/digiwf-engine-api-internal";
-import {EngineServiceApiConfig} from "../EngineServiceApiConfig";
+import {ApiConfig} from "../ApiConfig";
+import {PageOfTasks, TaskApiFactory, TasksApiFactory, TaskWithSchema} from "@muenchen/digiwf-task-api-internal"
 import {configuredAxios} from "../statusCodeHandling";
 
-export const callGetTasks = (page: number, size: number, query?: string, followUp?: boolean): Promise<PageHumanTaskTO> => {
-  const cfg = EngineServiceApiConfig.getAxiosConfig(FetchUtils.getGETConfig());
-  return getFactoryFromConfig(cfg).getTasks(page, size, query, followUp).then((res) => {
+/**
+ * old api for getting tasks. will be replaced by callGetTasksFromTaskService
+ * @deprecated
+ * @param page
+ * @param size
+ * @param query
+ * @param followUp
+ */
+export const callGetTasksFromEngine = (page: number, size: number, query?: string, followUp?: boolean): Promise<PageHumanTaskTO> => {
+  const cfg = ApiConfig.getAxiosConfig(FetchUtils.getGETConfig());
+  return HumanTaskRestControllerApiFactory(cfg).getTasks(page, size, query, followUp).then((res) => {
     return Promise.resolve(res.data);
   }).catch((err: any) => Promise.reject(FetchUtils.defaultCatchHandler(err, "Die Aufgaben konnten nicht geladen werden. Bitte versuchen Sie es erneut.")))
 };
+export const callGetTasksFromTaskService = (page: number, size: number, query?: string, followUp?: string): Promise<PageOfTasks> => {
+  // follow-up: YYYY-MM-dd: e.g. 2023-04-17
+  const cfg = ApiConfig.getTasklistAxiosConfig(FetchUtils.getGETConfig());
+  return TasksApiFactory(cfg).getCurrentUserTasks(page, size, query) // FIXME: followUp?
```

Comment:
https://app.zenhub.com/workspaces/digiwf-621f70bf50ea1100120b7e93/issues/gh/it-at-m/digiwf-core/318

Das müsste ja noch umgesetzt werden, oder